### PR TITLE
log actualValue regardless

### PR DESCRIPTION
**Pull request by mrdingma**
It would be helpful to show the actual value of the assertion regardless of whether it passed or failed.

### CODE REVIEW

**Review by mrdingma — COMMENTED** [review]
agree

File: gatling-core/src/main/resources/gatling-defaults.conf [hunk 1]
```diff
@@ -77,6 +77,7 @@ gatling {
     noReports = false       # When set to true, don't generate HTML reports
     maxPlotPerSeries = 1000 # Number of points per graph in Gatling reports
     useGroupDurationMetric = false  # Switch group timings from cumulated response time to group duration.
+    printActualValue = false   # When set to true, will print actual value of the assertion even if the assertion passes.
```
Comment (slandelle) [inline gatling-core/src/main/resources/gatling-defaults.conf:80]: No need for this option, let's print the actual value anyway

**Review by slandelle — CHANGES_REQUESTED** [review]
Thanks.
Could you please remove the option and sign our [CLA](https://docs.google.com/forms/d/1hbpxVqJ5hIYYJuOjOyOzub9fJFGFGvvfnB5w5LCWdUw/)?

Note: we'll start with the release process on Monday, so please hurry so this change doesn't miss it.

File: gatling-core/src/main/resources/gatling-defaults.conf (hunk 1 above)
Comment (mrdingma) [reply gatling-core/src/main/resources/gatling-defaults.conf]: sounds good. Updated!

File: gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala [hunk 2]
```diff
@@ -68,7 +68,7 @@ private final class RunResultProcessor(configuration: GatlingConfiguration) {
       assertionResult match {
         case AssertionResult.Resolved(_, success, actualValue) =>
           if (success) {
-            println(s"$message : true")
+              println(s"$message : true (actual : $actualValue)")
```
Comment (slandelle) [inline gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala:71]: format error :)

**Review by slandelle — CHANGES_REQUESTED** [review]

File: gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala (hunk 2 above)
Comment (mrdingma) [reply gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala]: ah ty. fixed

**Comment by slandelle** [comment]
Cherry-picked and clean up, thanks!

File: gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala [hunk 3]
```diff
@@ -68,7 +68,7 @@ private final class RunResultProcessor(configuration: GatlingConfiguration) {
       assertionResult match {
         case AssertionResult.Resolved(_, success, actualValue) =>
           if (success) {
-            println(s"$message : true")
+            println(s"$message : true (actual : $actualValue)")
           } else {
             println(s"$message : false (actual : $actualValue)")
           }
```
Comment (Isammoc) [inline gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala:74]: Now, same thing as:
```suggestion
          println(s"$message : $success (actual : $actualValue)")
```

Pedantic note: no space before semicolon in english:
```suggestion
          println(s"$message: $success (actual: $actualValue)")
```

**Pull request closed without merging**